requirements.txt

```diff
@@ -4,4 +4,5 @@ h5py
 colorlog
 colorama
 seqeval
-multiprocess
\ No newline at end of file
+multiprocess
+pre-commit
```

Comment:
Why adding this?

Reply:
Sorry,  actually I haven't see these two lines in my code file till now. Maybe I added these when I was using git bash commands. I'm a greenhand in github so I made mistakes.